packages/datagateway-download/src/downloadCart/urlBuilders/buildInvestigationUrl.test.ts

```diff
@@ -0,0 +1,171 @@
+import buildInvestigationUrl from './buildInvestigationUrl';
+import {
+  mockedSettings,
+  mockFacilityCycles,
+  mockInvestigations,
+} from '../../testData';
+import axios from 'axios';
+import type { Investigation } from 'datagateway-common';
+
+describe('buildInvestigationUrl', () => {
+  describe('given an investigation object', () => {
+    let investigation: Investigation;
+
+    beforeEach(() => {
+      investigation = mockInvestigations[0];
+      axios.get = jest.fn().mockResolvedValue({
+        data: [investigation],
+      });
+    });
+
+    it('should return a generic URL to it', async () => {
+      const url = await buildInvestigationUrl({
+        investigation,
+        apiUrl: mockedSettings.apiUrl,
+        facilityName: 'SVELTE',
+        facilityCycles: mockFacilityCycles,
```

Comment:
We don't need to pass this in in the tests anymore - leaving it here is potentially confusing